core/client/fs/src/test/java/alluxio/client/file/FileSystemTestUtils.java

```diff
@@ -71,7 +71,7 @@ public static void createByteFile(FileSystem fs, String fileName,
   public static void createByteFile(FileSystem fs, AlluxioURI fileURI,
       WritePType writeType, int len) {
     CreateFilePOptions options =
-        FileSystemClientOptions.getCreateFileOptions().toBuilder().setWriteType(writeType).build();
+        CreateFilePOptions.newBuilder().setRecursive(true).setWriteType(writeType).build();
```

Comment:
why is recursive set now?

Reply:
Alluxio Java API change. Explained in other comment.